cypress.config.ts

```diff
@@ -13,7 +21,192 @@ export default defineConfig({
     POSTGRES_URL: 'postgresql://postgres:postgres@localhost:5432/registry',
   },
   e2e: {
-    setupNodeEvents(on) {
+    setupNodeEvents(on, config) {
+      async function connectDB(query: string) {
+        const dbUrl = new URL(config.env.POSTGRES_URL);
+        const client = new pg.Client({
+          user: dbUrl.username,
+          password: dbUrl.password,
+          host: dbUrl.hostname,
+          database: dbUrl.pathname.slice(1),
+          port: Number(dbUrl.port),
+          ssl: false,
+        });
+        await client.connect();
+        const res = await client.query(query);
+        await client.end();
+        return res.rows;
+      }
+
+      on('task', {
+        connectDB,
+        async deleteUser(email = 'test@test.com') {
+          const [user] = await connectDB(`SELECT *
+                                          FROM users
+                                          WHERE email = '${email}';`);
```

Comment:
nit: style wise we can just have it all in one line
```suggestion
          const [user] = await connectDB(`SELECT * FROM users WHERE email = '${email}';`);
```